xenstoreclient/xenstore.go

```diff
@@ -461,9 +461,14 @@ func (xs *XenStore) StopWatch() error {
 	return nil
 }
 
+type ValueUpdate struct {
+	value string
+	update bool
+}
+
 type CachedXenStore struct {
 	xs         XenStoreClient
-	writeCache map[string]string
+	writeCache map[string]ValueUpdate
```

Comment:
Instread of ValueUpdate, I like to call it as Content and have value and keepalive
After UpdateCache, keepalive will be false.